ruby-ci/README.md

```diff
@@ -9,17 +9,17 @@ tools: ["Bundler", "RuboCop", "RSpec"]
 primary_emojis: [":ruby:"]
 ---
 
-# CI for Ruby with Bundler, Rubocop, and RSpec
+# CI for Ruby
 
-This template gives you a continuous integration (CI) pipeline that lints and tests a Ruby application.
+This template gives you a continuous integration (CI) pipeline that builds, lints, and tests a Ruby application.
 
 At a glance:
 
 - For [Ruby](https://www.ruby-lang.org/) applications
 - Uses [Bundler](https://bundler.io/)
 - Requires [Docker](https://docs.docker.com/get-docker/)
-- Runs [RuboCop](https://rubocop.org/) for linting
-- Runs [RSpec](https://rspec.info/) for testing
+- Lints code with [RuboCop](https://rubocop.org/)
```

Comment:
non-blocking: I wonder if we should follow the same "At a glance:" copywriting pattern that you used in the php-ci README, where the tools are prefixed with "Uses [TOOL_NAME]".

It seems verbose here because how the tool is used is mentioned both in "At a glance" and "How it works".

Maybe we could consider a bullet-point pattern like:

```
At a glance:

For [Ruby](https://www.ruby-lang.org/) applications using:
- [Bundler](https://bundler.io/)
- [RuboCop](https://rubocop.org/)
- [RSpec](https://rspec.info/)

Requires [Docker](https://docs.docker.com/get-docker/)

```

Reply:
non-blocking: Or, should we split out a 'prerequisites' section for templates that don't just work out of the box?

```
## Before you start

You need to have [Docker](https://docs.docker.com/get-docker/) running on your environment.

## At a glance

For [Ruby](https://www.ruby-lang.org/) applications using:
- [Bundler](https://bundler.io/)
- [RuboCop](https://rubocop.org/)
- [RSpec](https://rspec.info/)
```

Reply:
Both good suggestions. I'll make a card in our backlog with some of these thoughts for further discussion with @mbelton-buildkite and co.

Reply:
Both good suggestions. I'll make a card in our backlog with some of these thoughts for further discussion with @mbelton-buildkite and co.